styles.css

```diff
@@ -195,28 +184,33 @@ nav {
   border: none;
   width: 18%;
 }
-footer {
-  background-color: #5356A4;
-  display: flex;
-  justify-content: space-between;
-  align-items: stretch;
-  height: 2em;
-  border-top: 1px #1f1f3d solid;
+h3 {
+  font-weight: bold;
+  color: #1F1F3D;
 }
-.comment-button {
-  background: url(assets/comment.svg);
-  background-repeat: no-repeat;
-  width: 18%;
+p {
+  font-weight: normal;
+  padding-left: 1em;
+}
+.show-starred-button {
+  white-space: nowrap;
+  background-color: #5356A4;
+  font-size: bold;
+  color: #E9E9F3;
   border: 0;
+  border-radius: 2px;
+  padding: .5em 1.8em;
+  margin: 2em 1em;
 }
 .comment-label {
-    color: #E9E9F3;
-    font-weight: bold;
-    margin-top: .5em;
-    width: 82%;
-    height: 2.5em;
+  color: #E9E9F3;
+  font-weight: bold;
+  padding-bottom: 1em;
+  width: 82%;
+  height: 2.5em;
   border: 0;
 }
+/* ****** EXTRA SPECIFICS ******** */
 .hidden {
   display: none!important;
```

Comment:
So it is actually discouraged to use the `!important`. Are you getting the styling you need without it? If not, it might be a good idea to create a function in your js file that will apply it or remove it when necessary

Reply:
It works without it! Thank you!